shell/fsnode/fsnode.go

```diff
@@ -0,0 +1,10 @@
+/*
+Package fsnode is a high-level interface for filesystem nodes.
+
+Simple wrappers to:
+
+* create file and directory nodes from paths and readers,
+* extract file and directory nodes to your local filesytem, and
+* print file contents to writers.
+*/
+package fsnode
```

Comment:
On Wed, Apr 29, 2015 at 01:56:43PM -0700, W. Trevor King wrote:
> Wed, Apr 29, 2015 at 01:30:04PM -0700, Juan Batiz-Benet:
> > > +package fsnode
> > 
> > i think this package should be called `unixfs` as that's the
> > abstraction that this is calling to.
> 
> Will do, although I don't see what's especially Unix-y about these
> file nodes.

Done, although I still prefer fsnode or posixfs to unixfs.